sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/implementation/AzureSchemaRegistryBuilder.java

```diff
@@ -167,6 +167,19 @@ public AzureSchemaRegistryBuilder addPolicy(HttpPipelinePolicy customPolicy) {
         return this;
     }
 
+    private String apiVersion;
+
+    /**
+     * Sets the API version for the service.
+     *
+     * @param apiVersion The service API version.
+     * @return the AzureSchemaRegistryBuilder.
+     */
+    public AzureSchemaRegistryBuilder apiVersion(String apiVersion) {
```

Comment:
Should it accept `SchemaRegistryVersion` instead of string or should there be an overload that accepts it?